### ISSUE TITLE
Performance Issue At Launching on 3.1.x(stable) Unrelated with Plugins

### ISSUE THREAD

**Issue by Alalalalaki** [vercel/hyper#5775]
- [ ] I am on the [latest](https://github.com/vercel/hyper/releases/latest) Hyper.app version
- [ ] I have searched the [issues](https://github.com/vercel/hyper/issues) of this repo and believe that this is not a duplicate

- **OS version and name**: macOS 11.4
- **Hyper.app version**: 3.1.1
- **Link of a [Gist](https://gist.github.com/) with the contents of your .hyper.js**: 
- **Relevant information from devtools** _(CMD+ALT+I on macOS, CTRL+SHIFT+I elsewhere)_: 
- **The issue is reproducible in vanilla Hyper.app**: all plugins are removed

## Issue

After updating to 3.1.x(stable), the launch time of hyper becomes significantly longer. In particular, it now takes 4 seconds to show `$` while in 3.0.2(stable) it takes only 2 seconds, which is only slightly longer than iterm2 or original terminal.  And to be specific what seems to be slower now is the time between first jumping out the blue highlight and the following jumping out of `~ $`. 

![image](https://user-images.githubusercontent.com/31154974/126260192-3ec0c05f-ef77-4ba8-89f3-a1d8a4c53a30.png)

**Comment by nocturnalmonkey** [comment]
I've also noticed significant performance lags since updating yesterday. It's pretty unusable right now

**Comment by jesusiglesias** [comment]
> I've also noticed significant performance lags since updating yesterday. It's pretty unusable right now

Same here. Please, fix this loss of performance asap! Thanks

**Comment by id-clic** [comment]
Same here.

**Comment by walangitan** [comment]
Same issue on my Intel Mac. In the meantime, downgrading to the previous version I had restores performance for me.

**Comment by LabhanshAgrawal** [comment]
Can you all try out the build from https://github.com/vercel/hyper/actions/runs/1049653907 to see if there's any improvement in perf.

**Comment by walangitan** [comment]
I'm seeing improved performance, similar to the previous version, from https://github.com/vercel/hyper/actions/runs/1049653907 using https://github.com/vercel/hyper/suites/3286035188/artifacts/76667686.

**Comment by Alalalalaki** [comment]
@LabhanshAgrawal Yes, it improves the launch performance, but still 0.5s slower than 3.0.2 in my computer.

**Comment by marcofugaro** [comment]
@walangitan how do you downgrade hyper?

**Comment by acourtiol** [comment]
I've also the issue on macOS 11.4 and two MBP 13" 2020 (Intel and Apple Silicon).
Tried out the build from https://github.com/vercel/hyper/actions/runs/1049653907 without any improvements, switching to iTerm2 until a fix is available

**Comment by marcofugaro** [comment]
How to downgrade: download and install the old version at https://github.com/vercel/hyper/releases/tag/3.0.2
Open Hyper, DO NOT restart it after, as it will install the newer version.

This sucks, please provide an option to disable auto-updates or republish the 3.0.2 as latest.

**Comment by marcofugaro** [comment]
cc @rauchg can you fix this?

**Comment by stephenchaffins** [comment]
Same issue on my Mac as well. Removing all plugins helped, but its still insanely slow.  MacOS 11.0.1 on a 2015 MB Pro

**Comment by nocturnalmonkey** [comment]
> Can you all try out the build from https://github.com/vercel/hyper/actions/runs/1049653907 to see if there's any improvement in perf.

This seems to have fixed the issue for me

**Comment by 201flaviosilva** [comment]
I have the same problem, I can barely write anything, the hyper is very slow now, it lost a lot of performance since the last update, please fix it :)

**Comment by abotts** [comment]
Performance so bad I went back to using the default terminal :(

**Comment by tiomno** [comment]
> https://github.com/vercel/hyper/actions/runs/1049653907

This fixed the issue for me, thanks @LabhanshAgrawal

**Comment by mikegarite** [comment]
Still major issues with every upgrade. Performance is super sludge.

**Comment by rhummelmose** [comment]
Still issue in 3.1.1

**Comment by jak-kal** [comment]
Absolutely dog-slow. Cannot use this any more. Been using since I moved to my Mac (almost 2+ years ago).
